projects/optic/src/init.ts

```diff
@@ -71,5 +76,12 @@ export const initCli = async () => {
   registerRulesetUpload(rulesetSubcommands, cliConfig);
   registerRulesetInit(rulesetSubcommands, cliConfig);
 
+  // commands for tracking changes with openapi
+  cli.addCommand(await captureCommand());
+  cli.addCommand(await newCommand());
+  cli.addCommand(await captureCertCommand());
+  cli.addCommand(await clearCommand());
+  cli.addCommand(await verifyCommand());
```

Comment:
Do we want these to be under some sort of subcommand? Do you think it'll clash with any of the existing commands / commands we're planning to build?

Reply:
I actually think `optic oas [...]` isn't bad. I'll make that update